src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.spec.ts

```diff
@@ -221,26 +231,9 @@ describe('DotEditPageToolbarComponent', () => {
         expect(component.onLockerClick).toHaveBeenCalledTimes(1);
     });
 
-    it('should NOT have an action split button', () => {
-        const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__actions'));
-        expect(primaryAction).toBeFalsy();
-    });
-
     it('should have an action split button', () => {
-        component.pageWorkflows = [{ name: 'Workflow 1', id: 'one' }, { name: 'Workflow 2', id: 'two' }];
-        fixture.detectChanges();
         const primaryAction: DebugElement = de.query(By.css('.edit-page-toolbar__actions'));
-        expect(primaryAction).toBeTruthy();
-        expect(primaryAction.name).toEqual('p-splitButton', 'is a splitbutton');
-    });
-
-    it('should set action split buttons params', () => {
-        component.pageWorkflows = [{ name: 'Workflow 1', id: 'one' }, { name: 'Workflow 2', id: 'two' }];
-        fixture.detectChanges();
-        const actionsButton: SplitButton = de.query(By.css('.edit-page-toolbar__actions')).componentInstance;
-
-        expect(actionsButton.label).toEqual('Acciones', 'actions label is set');
-        expect(actionsButton.model).toEqual([{ label: 'Workflow 1' }, { label: 'Workflow 2' }]);
+        expect(primaryAction === null).toBe(false);
```

Comment:
This test it's not ok, because you are querying the node element by ```.edit-page-toolbar__actions``` but the toolbar can't know the the ```.edit-page-toolbar__actions``` in inside a node element of the component ```<dot-edit-page-workflows-actions>```, you should do the query by the direct tag that the toolbar have in the HTML: ```dot-edit-page-workflows-actions```

Reply:
you should use a mock of dot-edit-page-workflows-actions component, also we shoukd check if the component has the right values and not just check if it is there